docs/scarpet/Full.md

```diff
@@ -3995,6 +3995,10 @@ at the same time.
 Triggered right after a lightning strikes. Lightning entity as well as potential horseman trap would 
 already be spawned at that point. `mode` is `true` if the lightning did cause a trap to spawn. 
 
+### `__on_explosion(pos, power, source, create_fire, blocks)`
+Triggered right before an explosion blows up, right before any changes to the world are done, but the decision to blow up is already made.  
+The parameter `blocks` contains the list of block that will blow up (or would blow up if `explosionNoBlockDamage` is setted to `true`).
+
```

Comment:
This is not needed due to GitHub Actions automatically merging any documentation changes to the topic documentation files into Full.md.

Reply:
Wow! I've never heard about GitHub Action before!
Nice to know!